man/systemd-system.conf.xml

```diff
@@ -31,7 +31,9 @@
     <filename>/etc/systemd/system.conf.d/*.conf</filename>,
     <filename>/run/systemd/system.conf.d/*.conf</filename>,
     <filename>/usr/lib/systemd/system.conf.d/*.conf</filename></para>
-    <para><filename>/etc/systemd/user.conf</filename>,
+
+    <para><filename>$XDG_CONFIG_HOME/systemd/user.conf</filename>,
+    <filename>/etc/systemd/user.conf</filename>,
```

Comment:
hmm, what? we didn't read ~/.config/systemd/user.conf so far? ouch, that's a bug